features/administration.feature

```diff
@@ -59,3 +59,32 @@ Feature: Bookstore Administration Panel
     Then I see the book "Book That Has Been Edited"
     When I visit the public book index
     Then I see the book "Book That Has Been Edited"
+
+  Scenario: Viewing orders
+    Given I am logged into the admin panel
+    And there are 100 books in the database
+    And some books have been ordered
+    When I visit the admin orders url
+    Then I see the orders
+
+  Scenario: Adding a book discount
+    Given I am logged into the admin panel
+    And I am logged into the site
+    And there is a book named "Book To Be Discounted"
+    And the book is valued at "$20.00"
```

Comment:
Cool. Yea, we're not using that syntax anywhere yet, but I like it better when you're detailing out the attributes for an object.